cmd/server/main.go

```diff
@@ -301,6 +302,29 @@ func realMain(ctx context.Context) error {
 		sub.Handle("/{uuid}/expire", codeStatusController.HandleExpirePage()).Methods("PATCH")
 	}
 
+	// mobileapp
+	{
+		sub := r.PathPrefix("/mobileapp").Subrouter()
+		sub.Use(requireAuth)
+		sub.Use(loadCurrentRealm)
+		sub.Use(requireRealm)
+		sub.Use(processFirewall)
+		sub.Use(requireAdmin)
+		sub.Use(requireVerified)
+		sub.Use(requireMFA)
+		sub.Use(rateLimit)
+
+		mobileController := mobileapp.New(ctx, config, cacher, db, h)
+		sub.Handle("", mobileController.HandleIndex()).Methods("GET")
+		sub.Handle("", mobileController.HandleCreate()).Methods("POST")
+		sub.Handle("/new", mobileController.HandleCreate()).Methods("GET")
+		sub.Handle("/{id}/edit", mobileController.HandleUpdate()).Methods("GET")
```

Comment:
Add constraints on these IDs (we haven't backported to the other things yet), but they should be:

```
{id:[0-9]+}
```